### PR TITLE
TST: Added UnitTests for Matchers.java & ByteArrayMatchers.java

### PR DESCRIPTION
**Pull request by SimonaStavarache**

### CODE REVIEW

**Review by austin0 — DISMISSED** [review]

File: yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        Description testDescription = new TestDescription();
+
+        baseMatcherObject.describeTo(testDescription);
+        assertThat(testDescription.toString(), is("\n\t\tae"));
+    }
+
+    @Test
+    void testMatchesHex() {
+        Matcher<byte[]> baseMatcherObject = matchesHex("0c0d647F0c");
+        assertTrue(baseMatcherObject.matches(new byte[]{12, 13, 100, 127, 0xc}));
```
Comment (joe-chacko) [inline yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java]: Use `new byte[]{0x0c, 0x0d, 0x64, 0x7f, 0x0c}`

File: yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    @Test
+    void testNotMatchesHex() {
+        Matcher<byte[]> baseMatcherObject = matchesHex("0c0d647F0c");
+        assertFalse(baseMatcherObject.matches(new byte[]{12, 13, 100}));
```
Comment (joe-chacko) [inline yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java]: use hex int literals

File: yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        baseMatcherObject.describeTo(testDescription);
+        assertThat(testDescription.toString(), is("\n\t\t0c0d647F"));
+    }
+
+    @Test
+    void testMismatchDescription() {
+        Matcher<byte[]> baseMatcherObject = matchesHex("0c0d647F");
+
+        Description testDescription = new TestDescription();
+        baseMatcherObject.describeMismatch(new byte[]{12, 13, 100}, testDescription);
```
Comment (joe-chacko) [inline yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java]: use hex int literals

**Review by joe-chacko — CHANGES_REQUESTED** [review]
See comments for details

File: yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java (hunk 1 above)
Comment (SimonaStavarache) [reply yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java]: Solved

File: yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java (hunk 2 above)
Comment (SimonaStavarache) [reply yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java]: Solved

File: yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java (hunk 3 above)
Comment (SimonaStavarache) [reply yoko-testify/src/test/java/testify/matchers/ByteArrayMatchersTest.java]: Solved

**Review by austin0 — APPROVED** [review]

**Comment by SimonaStavarache** [comment]
Closed #179

**Pull request merged**